synapse/lib/stormtypes.py

```diff
@@ -2077,10 +2077,14 @@ def _parseAlias(self, opts):
             incunit = valinfo[requnit][1]
             incval = 1
 
-        # Remove the curly braces
-        query = query[1:-1]
+        cdef = {'storm': query[1:-1],
+                'reqs': reqdict,
+                'incunit': incunit,
+                'incvals': incval,
+                'useriden': self.runt.user.iden
+                }
 
-        todo = s_common.todo('addCronJob', self.runt.user.iden, query, reqdict, incunit, incval)
+        todo = s_common.todo('addCronJob', cdef)
         gatekeys = ((self.runt.user.iden, ('cron', 'add'), None),)
         iden = await self.dyncall('cortex', todo, gatekeys=gatekeys)
```

Comment:
Lets make the addCronJob API return the new cdef ( to align with the other stormtypes uses )